shared/modules/consensus_module.go

```diff
@@ -74,6 +73,8 @@ type ConsensusPacemaker interface {
 type ConsensusStateSync interface {
 	GetNodeIdFromNodeAddress(string) (uint64, error)
 	GetNodeAddress() string
+	IsSynched() (bool, error)
+	GetValidatorsAtHeight(uint64) ([]*types.Actor, error)
 }
 
 type ConsensusDebugModule interface {
```

Comment:
Can you add some comments on where/how the debug interface is intended to be used.